src/Deserialization/Transaction.js

```diff
@@ -59,6 +59,9 @@ exports._txBodyAuxiliaryDataHash = maybeGetter("auxiliary_data_hash");
 exports._txBodyValidityStartInterval = maybeGetter("validity_start_interval_bignum");
 // multiassets(): Mint | void;
 exports._txBodyMultiAssets = maybeGetter("multiassets");
+exports._txBodyReferenceInputs = maybe => containerhelper =>
+    body => body.reference_inputs() ?
```

Comment:
I think this formatting is a bit hard to read, but that might just be me

Reply:
Speaking of which we should probably add `prettier` or something and format all of the JS (along with a linter). Unrelated to this PR of course

Reply:
https://github.com/Plutonomicon/cardano-transaction-lib/issues/530